.release-please-manifest.json

```diff
@@ -13,5 +13,5 @@
   "libs/providers/ofrep-web": "0.1.3",
   "libs/providers/flipt": "0.1.0",
   "libs/providers/flagsmith-client": "0.1.2",
-  "libs/providers/flipt-web": "0.1.0"
```

Comment:
can we force it to be `0.1.0` @beeme1mr ?

Reply:
There's a `release-as` setting but I've never used it before.

Reply:
If you really care for a 0.1.0, you can add the 'release-as' in the config object

Reply:
I've done this.

cc @beeme1mr @markphelps